stdpopsim/slim_engine.py

```diff
@@ -625,41 +638,28 @@ def script_file_f():
 
             ts = pyslim.load(ts_file.name)
 
-        # random.seed(seed)
-        # s1, s2 = random.randint(1,2**32-1), random.randint(1,2**32-1)
+        if do_recap:
+            random.seed(seed)
```

Comment:
Should create our own instance of RNG here rather than modifying the global state, ``rng = random.Random(seed)``.

Reply:
Done. I also changed `randint(1, 2**32-1)` to the much clearer `randrange(2**32)`.